### PR TITLE
Report `braid-design-system` version via telemetry

### PR DESCRIPTION
**Pull request by askoufis**
I initially tried getting the braid version from the app's `package.json` dependencies, but the version _could_ be a caret version (shows up as `_32.0.0` in datadog), for example in a library, so I instead opted to fetch it from the version in the app's `node_modules`.

### CODE REVIEW

File: packages/sku/telemetry/index.js [hunk 1]
```diff
@@ -1,24 +1,32 @@
...
 try {
   const packageJson = require(getPathFromCwd('package.json'));
 
   if (packageJson.name) {
     projectName = packageJson.name;
   }
+
+  const braidPackageJson = require(require.resolve(
+    'braid-design-system/package.json',
+  ), [cwd()]);
```
Comment (mrm007) [inline packages/sku/telemetry/index.js:21]:
```suggestion
  const braidPackageJson = requireFromCwd('braid-design-system/package.json');
```

👀 

https://github.com/seek-oss/sku/blob/721990812bb10d5c5879d19e15af29b652c5bbf7/packages/sku/lib/cwd.js#L25-L29

File: packages/sku/telemetry/index.js [hunk 2]
```diff
@@ -1,24 +1,32 @@
 const os = require('node:os');
 
-const { getPathFromCwd } = require('../lib/cwd');
+const { getPathFromCwd, cwd } = require('../lib/cwd');
```
Comment (mrm007) [inline packages/sku/telemetry/index.js:3]:
```suggestion
const { requireFromCwd } = require('../lib/cwd');
```

File: packages/sku/telemetry/index.js [hunk 3]
```diff
@@ -1,24 +1,32 @@
...
 const isCI = require('../lib/isCI');
 const provider = require('./provider');
 const skuVersion = require('../package.json').version;
 
 const { languages } = require('../context');
 
 let projectName = 'unknown';
+let braidVersion = 'unknown';
 try {
   const packageJson = require(getPathFromCwd('package.json'));
```
Comment (mrm007) [inline packages/sku/telemetry/index.js:13]:
```suggestion
  const packageJson = requireFromCwd('package.json');
```

File: packages/sku/telemetry/index.js [hunk 4]
```diff
@@ -1,24 +1,32 @@
...
   if (packageJson.name) {
     projectName = packageJson.name;
   }
+
+  const braidPackageJson = require(require.resolve(
+    'braid-design-system/package.json',
+  ), [cwd()]);
+
+  braidVersion = braidPackageJson.version;
 } catch (e) {}
```
Comment (mrm007) [inline packages/sku/telemetry/index.js:24]: Can we add a `require('debug')('sku:telemetry')` message here?

Comment (askoufis) [reply packages/sku/telemetry/index.js:24]: For the `catch`? Sure. The telemetry package already spits out debug logs for the stuff it handles. https://github.com/SEEK-Jobs/sku-telemetry/pull/17

Comment (mrm007) [reply packages/sku/telemetry/index.js:24]: I meant on the sku side, so we see the error if any of the `package.json`s can't be loaded.

File: packages/sku/telemetry/index.js (hunk 1 above)
Comment (askoufis) [reply packages/sku/telemetry/index.js]: Ooo, should've scouted this out, felt like I was re-implementing something that already existed.

**Review by mrm007 — APPROVED** [review]

**Pull request merged**